### ISSUE TITLE
Question regarding the network model

### ISSUE THREAD

**Issue by PokerBingo** [dewenzeng/effort_prediction_mce#1]
Hello!

I was trying to replicate the model in the paper and I am having some difficulty doing so.

The paper claims that GTX 1080Ti GPUs were used, but with this dataset of 512x512 images, I am running out of GPU memory when running the model. I also have GTX 1080Ti GPUs. That is where my question comes in.

Do you never downsample (like, maxpool) the images from one residual block to the next? That is, are the images propagating through the network always with size 512x512?

![model](https://user-images.githubusercontent.com/45846847/218997154-60bc5404-86a6-4a8a-83ba-a4f052cb925b.png)

Also, as you can see above, is there no activation after the first convolution layer?

Finally, the attention block starts with a maxpool operation. In total, 3 maxpools are done, but only 2 upsamples are performed. This means that the output of the attention block will be smaller than its input. When adding the output of the attention block with the output of the residual blocks, the sizes are not the same and I get an error.

In short, could you provide more explicit information on how exactly your model was constructed?

Thanks in advance.
